packages/material/scss/menu/_layout.scss

```diff
@@ -60,8 +61,11 @@ $item-selected-opacity: .16 !default;
     .k-menu-scroll-wrapper {
 
         .k-menu-scroll-button {
-            padding: $button-padding-x;
             box-shadow: none;
         }
+
+        &:not(.vertical) .k-menu-scroll-button {
```

Comment:
I was puzzled why this rule applies, given the `vertical` class name. I believe this class should be renamed by convention.

Reply:
I have logged [an issue](https://github.com/telerik/kendo/issues/8392) to fix the class name by convention. For this moment this class is used also in LESS themes.